include/aolayer.h

```diff
@@ -180,6 +180,7 @@ class CharLayer : public AOLayer {
   void play_frame_effect(int p_frame);
 
 private slots:
+  void preanim_done(); // overloaded so we don't accidentally cull characters
   void movie_ticker(); // overloaded so we can play effects
```

Comment:
C++ tip: add `override` to functions you want to override, to ensure that an overload is not being accidentally made.

Reply:
done, see https://github.com/AttorneyOnline/AO2-Client/pull/302/commits/f4cbdd23a72ff3bcbd7dc53e34d4be6af12e1839